rust_src/src/eval.rs

```diff
@@ -14,11 +10,21 @@ fn xsignal(error_symbol: LispObject, data: LispObject) -> ! {
     unsafe {
         Fsignal(error_symbol.to_raw(), data.to_raw());
     }
-
-    unreachable!();
 }
 
 /// Convenience function for calling `xsignal` with an empty list.
 pub fn xsignal0(error_symbol: LispObject) -> ! {
     xsignal(error_symbol, Qnil);
 }
+
+/// Convenience function for calling `xsignal` with a one-element list.
+// pub fn xsignal1(error_symbol: LispObject, arg: LispObject) -> ! {
```

Comment:
It's currently not used - I could remove it or add a `allow(dead_code)` instead.